src/icons/fileIcons.ts

```diff
@@ -885,7 +885,7 @@ export const fileIcons: FileIcons = {
         '.posthtmlrc.yml',
       ],
     },
-    { name: 'todo', fileExtensions: ['todo'] },
+    { name: 'todo', fileExtensions: ['todo'], fileNames:['TODO.md', 'todo.md'] },
```

Comment:
Nice idea. Could you please remove the uppercase "TODO.md" as the fileNames are case insensitive. So the uppercase version is included when we already have a lowercase file name. In addition, what do you think about the plural, "todos.md", shall we add this as well?

Reply:
yea ill make a new pull request